gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-all.zip
```

Comment:
This pages displays differences in the files between our repo and your PR branch. Since we want to keep this file unchanged, I would copy file content from repository and paste, so that there won't be any difference, and file will not appar here :)